tests/dummy/app/pods/link/template.hbs

```diff
@@ -1,292 +1,292 @@
-            </div>
-            <div class="snippet">
-              {{format-markdown "```handlebars
-{{#frost-link
-  'link.min'
-  priority='primary'
-  size='small'
-  disabled=true
-}}
-  <div class='text'>Primary</div>
-{{/frost-link}}
-```"
-              }}
-            </div>
-        </div>
+  <div class="section-title">Primary</div>
 
-        <div class="example">
-            <div class="title">Small</div>
-            <div class="demo">
-              {{#frost-link
-                'link.min'
-                priority='primary'
-                size='small'
-              }}
-                  <div class='text'>Primary</div>
-              {{/frost-link}}
-            </div>
-            <div class="snippet">
-              {{format-markdown "```handlebars
-{{#frost-link
-  'link.min'
+  <div class="section">
+
+    <div class="example">
+      <div class="title">Small</div>
+      <div class="demo">
+        {{#frost-link 'link.min'
+          priority='primary'
+          size='small'
+        }}
+          Primary
+        {{/frost-link}}
+      </div>
+      <div class="snippet">
+        {{format-markdown "```handlebars
+{{#frost-link 'link.min'
   priority='primary'
   size='small'
 }}
-  <div class='text'>Primary</div>
+  Primary
 {{/frost-link}}
-```"
-              }}
-            </div>
-        </div>
+```"}}
+      </div>
+    </div>
 
-        <div class="example">
-            <div class="title">Medium</div>
-            <div class="demo">
-              {{#frost-link
-                'link.min'
-                priority='primary'
-                size='medium'
-              }}
-                  <div class='text'>Primary</div>
-              {{/frost-link}}
-            </div>
-            <div class="snippet">
-              {{format-markdown "```handlebars
-{{#frost-link
-  'link.min'
+    <div class="example">
+      <div class="title">Medium</div>
+      <div class="demo">
+        {{#frost-link 'link.min'
+          priority='primary'
+          size='medium'
+        }}
+          Primary
+        {{/frost-link}}
+      </div>
+      <div class="snippet">
+        {{format-markdown "```handlebars
+{{#frost-link 'link.min'
   priority='primary'
   size='medium'
 }}
-  <div class='text'>Primary</div>
+  Primary
 {{/frost-link}}
-```"
-              }}
-            </div>
-        </div>
+```"}}
+      </div>
+    </div>
 
-        <div class="example">
-            <div class="title">Large</div>
-            <div class="demo">
-              {{#frost-link
-              'link.min'
-              priority='primary'
-              size='large'
-              }}
-                  <div class='text'>Primary</div>
-              {{/frost-link}}
-            </div>
-            <div class="snippet">
-              {{format-markdown "```handlebars
-{{#frost-link
-  'link.min'
+    <div class="example">
+      <div class="title">Large</div>
+      <div class="demo">
+        {{#frost-link 'link.min'
```

Comment:
I'd like to see the small/medium/large in one example/demo combo so that all three demos can be shown side by side horizontally, i.e.

Code small
Code medium
Code large
Small demo - Medium demo - Large demo